tests/unit/test_prerequisite.py

```diff
@@ -138,14 +143,79 @@ def test_get_target_points(prereq):
     }
 
 
-def test_get_resolved_dependencies():
+@pytest.fixture
+def prereq2():
```

Comment:
suggest `integer_prereq` for this one and `iso8601_prereq` for the other pre-req fixture.

Or shove them in a singe fixutre thus:

```python
@pytest.fixture
def prereq():
    def _inner(iso8601: boolean = False):
        if iso8601:
            # setup prereq 1
            return prereq
        else:
            # setup prereq 2
            return prereq
    return _inner
```

Reply:
I think it makes sense to have this fixture here as it is visible while you are looking at the 2 tests that use it